### PR TITLE
Add karambwans to cooking plugin.

### PR DESCRIPTION
**Pull request by Monsterxsync**
Karambwans are missing in the cooking plugin this adds them to it.

### CODE REVIEW

**Comment by Monsterxsync** [comment]
more of a bug than a feature request lol

File: runelite-client/src/main/java/net/runelite/client/plugins/cooking/CookingPlugin.java [hunk 1]
```diff
@@ -109,7 +109,11 @@ public void onChatMessage(ChatMessage event)
 
 		final String message = event.getMessage();
 
-		if (message.startsWith("You successfully cook") || message.startsWith("You successfully bake") || message.startsWith("You manage to cook") || message.startsWith("You roast a"))
+		if (message.startsWith("You successfully cook") ||
+			message.startsWith("You successfully bake") ||
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/cooking/CookingPlugin.java]: Other multiline conditionals in this project are typically styled:

```
if (a
    || b
    || c)
{
    // ...
}
```

I'd  recommend using that style here as well.

Comment (Monsterxsync) [reply runelite-client/src/main/java/net/runelite/client/plugins/cooking/CookingPlugin.java]: yeah, just changed it to that thinking it was the checkstyle error but thats not it. :/

**Comment by Adam-** [comment]
Can someone get a full list of all of the full cooking messages?

**Comment by Monsterxsync** [comment]
Ill have a look and see if anything else is broken, i know that you "cook" seaweed into soda ash but it gives no xp so the calc for cooked per hour wouldnt work.

**Comment by Adam-** [comment]
Can you get the messages though? I wanted to write a test.

**Comment by Monsterxsync** [comment]
Not unless theres an easy way to dump them took ages just going through what you can buy on ge lol

**Pull request merged**